src/style.css

```diff
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  margin: 0;
+}
+
+.form-field {
+  position: relative;
+  display: block;
+  margin: 20px 8px;
+}
+
+input {
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
```

Comment:
Always set box-sizing in the `* {...}` selector - Read more: https://css-tricks.com/box-sizing/